canonical-web-and-design/ubuntu.com
# Observability managed build

## Done

- Built new `/observability/managed` page as per [copy doc](https://docs.google.com/document/d/1N11otwvy2TWoK4j9e8iYJLNLYQyWbIPL82GVkm8WHhE/edit#heading=h.mhbttlbulk4)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-10862.demos.haus/observability/managed
- Compare against copy doc 
- Test ctas and ensure that they and the forms they link to work correctly

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4578
https://github.com/canonical-web-and-design/web-squad/issues/4579
https://github.com/canonical-web-and-design/web-squad/issues/4581

## Screenshots

[If relevant, please include a screenshot.]
